src/models/pod.js

```diff
@@ -1,47 +1,44 @@
-let users = {
-  1: {
-    id: "1",
-    username: "Robin X",
-  },
-  2: {
-    id: "2",
-    username: "Carol S",
-  },
-};
 
-let messages = {
-  1: {
-    id: "1",
-    text: "Good Day",
-    userId: "1",
-  },
-  2: {
-    id: "2",
-    text: "By World",
-    userId: "2",
-  },
-};
+const { Sequelize, Model, DataTypes } = require('sequelize');
+const sequelize = new Sequelize('sqlite::memory:');
 
-// const mongoose = require("mongoose");
-// const Schema = mongoose.Schema;
+class Pod extends Model {}
+Pod.init({
+  id: DataType.STRING,
+  podname: DataType.STRING
+  rating: DataType.STRING,
+}, { sequelize, modelName: 'pod' });
 
-// const itemsSchema = new Schema({
-//   user: {
-//     user: { type: Schema.Types.ObjectId, ref: "User" },
-//   },
+(async () => {
+  await sequelize.sync();
```

Comment:
This probably should be up in the main index.js to run once for all the models once they're defined. I think.